### ISSUE TITLE
(electron) 'isMenuBarAutoHide function' is deprecated and will be removed.

### ISSUE THREAD

**Issue by lutzroeder** [electron/electron#20698]
### Issue Details

* **Electron Version:** v7.0.0
* **Operating System:** Windows 10.0.18362.418
* **Last Known Working Electron version:** 6.x

### Expected Behavior
`npx electron .` runs without printing to console.

### Actual Behavior
`npx electron .` prints `(electron) 'isMenuBarAutoHide function' is deprecated and will be removed.` to console even though this API is not being called in user code.

### To Reproduce
```
git clone https://github.com/lutzroeder/netron.git
npm install
npx electron .
[ click "Open Model..." button ]
```

### Additional Information

Happens on Windows but not on macOS.

**Comment by davidjconnelly** [comment]
Hi.  I do not have a fix but I can confirm that I'm experiencing the same problem.  I think this must be an issue with v7.  Do you think going back to an older version of Electron might be a good idea?

**Comment by monsterkodi** [comment]
This is what I get on every startup:
```
(electron) 'getName function' is deprecated and will be removed. Please use 'name property' instead.
(electron) 'setName function' is deprecated and will be removed. Please use 'name property' instead.
(electron) 'isMenuBarAutoHide function' is deprecated and will be removed. Please use 'autoHideMenuBar property' instead.
```

**Comment by miniak** [comment]
I don't see `app.getName()` in `app.setName()` being called in Electron code.

**Comment by miniak** [comment]
https://github.com/electron/electron/blob/b6246dcf122e584c672566877cb60d33dbc4705e/lib/browser/api/browser-window.js#L36

**Comment by yafp** [comment]
I get similar warnings 

```
(electron) 'getName function' is deprecated and will be removed. Please use 'name property' instead.
(electron) 'isMinimizable function' is deprecated and will be removed. Please use 'minimizable property' instead.
(electron) 'isMaximizable function' is deprecated and will be removed. Please use 'maximizable property' instead.
(electron) 'isClosable function' is deprecated and will be removed. Please use 'closable property' instead.
```

with electron 7.0.1

**Comment by mtasir** [comment]
I still get the same warning in Windows 7 :
`(electron) 'isMenuBarAutoHide function' is deprecated and will be removed. Please use 'autoHideMenuBar property' instead.`

**Comment by yafp** [comment]
Should that now be fixed with 7.1.2?
I am asking as i just jumped from 7.1.1 to 7.1.2 and still got the issue

**Comment by yafp** [comment]
Additional info:
my warning appear in an Ubuntu environment. So i don't think its a windows platform specific thing (see issue labels)
